### PR TITLE
Templates comply with Octane ember-template-lint rules

### LINKED ISSUE #171: Update templates to comply with "octane" lint rules
This should be done in separate PR, there's a lot of template issues.

_Originally posted by @raido in https://github.com/adopted-ember-addons/ember-page-title/pull/170#r476253352_

### PR DESCRIPTION
**Pull request by raido**
This PR does bare minimum to get ember-tempate-lint pass. We also turned off `no-actions` rule for one component until is migrated to Glimmer component.

Closes #171.

### CODE REVIEW

File: tests/dummy/app/index/template.hbs [hunk 1]
```diff
@@ -18,28 +18,27 @@
...
       </div>
     </section>
 
     <section id="playground">
       <h2>Playground</h2>
       <div class="grid">
-        {{#page-title-pane as |p|}}
-          {{#p.matryoshka items=model.tokens as |token|}}
+        <PageTitlePane as |p|>
+          <p.matryoshka @items={{this.model.tokens}} as |token|>
```
Comment (raido) [inline tests/dummy/app/index/template.hbs:31]: `@model` become available in Ember.js v3.14+ and we are targeting v3.12, so has to be `this.model` for now.

File: yarn.lock [hunk 2]
```diff
@@ -4943,7 +4943,7 @@ debug@~4.1.0:
   dependencies:
     ms "^2.1.1"
 
-debuglog@*, debuglog@^1.0.1:
+debuglog@^1.0.1:
```
Comment (raido) [inline yarn.lock:4946]: It seems master branch has yarn.lock a bit out of date. 🤔

File: tests/dummy/app/index/template.hbs (hunk 1 above)
Comment (knownasilya) [reply tests/dummy/app/index/template.hbs:31]: Actually I think the latest ember-cli-update drops 3.12, but I'm unsure if the title RFC says 3.12+ or not.

Comment (raido) [reply tests/dummy/app/index/template.hbs:31]: RFC does not mention version. What I meant here is that current Travis setup in this repo targets v3.12.

Comment (knownasilya) [reply tests/dummy/app/index/template.hbs:31]: Maybe we should run ec-update again to get the latest support matrix.

Comment (raido) [reply tests/dummy/app/index/template.hbs:31]: We have all LTS versions from 3.12 to 3.20 included.

Comment (knownasilya) [reply tests/dummy/app/index/template.hbs:31]: 3.12 was dropped in the latest new-addon output: https://github.com/ember-cli/ember-addon-output/tree/v3.21.2 It's now 3.16 +

Comment (raido) [reply tests/dummy/app/index/template.hbs:31]: Yeah, sure. I would do the 3.12 drop in an another PR. Bumping dependencies and so on.

**Pull request merged**